tests/test_lockFile.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+import os
+from meg_runtime.locking.lockFile import LockFile
+
+
+
+@pytest.fixture()
+def generateLockfile():
+    fileName = ".meg/templockfile"
+    try:
```

Comment:
You don't want to catch errors in the test. They should be emitted, failing the test. You should have caught the exceptions in the code.